cmd/openshift-install/testdata/imagebased/configimage/manifests/extra_manifests.txt

```diff
@@ -0,0 +1,66 @@
+# Verify that the extra-manifests directory populates with the appropriate files
+
+exec openshift-install image-based create config-image --dir $WORK
+
+exists $WORK/imagebasedconfig.iso
+exists $WORK/auth/kubeconfig
```

Comment:
We can get rid of the checks for the `auth/` and the `/cluster-configuration/manifest.json`, as those are tested in the `assets/default_assets.txt` integration test.